docs/Installation.md

```diff
@@ -31,6 +32,22 @@ Docker are found there.
 The rest of this document is about doing a local installation of Zonemaster-CLI,
 not relevant for running Zonemaster-CLI on Docker.
 
+## Pre-built packages
+
+Pre-built packages are available for Debian and Ubuntu (architecture amd64 only).
+
+> Note: In order to use algorithm 15 and 16 on Ubuntu 20.04 the CPAN installation
+> must be used.
+
+1) Configure Zonemaster repository
+   ```sh
+   curl -Ls https://package.zonemaster.net/setup.sh | sudo bash
+   ```
+2) Install Zonemaster CLI
+   ```sh
+   sudo apt install zonemaster-cli
+   ```
+
```

Comment:
1. Will the package also update the locale setting?
2. Without support for algorithms 15 and 16 Zonemaster is incomplete. We should not have instructions for installing an incomplete Zonemaster. Furthermore, we state in the main README that there is support for algorithms 15 and 16 on Ubuntu 20.04.
3. I do not see why this is put in a separate section. Instead this should be the installation instruction for Debian.
4. The installation instruction for Engine for Debian should also be updated. Else you will first install Zonemaster::LDNS and Zonemaster::Engine with CPAN and come here to install the same thing with package.

Reply:
> The installation instruction for Engine for Debian should also be updated. Else you will first install Zonemaster::LDNS and Zonemaster::Engine with CPAN and come here to install the same thing with package.

There is technically no need to go to install the engine / ldns first as it will be installed as dependency of the cli. I will update of users wanting the engine only after this PR as been done (the changes are very similar so once this one is resolved I can port them to engine)

> I do not see why this is put in a separate section. Instead this should be the installation instruction for Debian.

That is what I did first but then the instruction for packages are after the Prerequisites section even though there is no Prerequisites. So to make it clearer I preferred having two sections. Also this way it is clearer that a new installation method is available. I can put it back to the other section if you think that is better.

> Without support for algorithms 15 and 16 Zonemaster is incomplete. We should not have instructions for installing an incomplete Zonemaster. Furthermore, we state in the main README that there is support for algorithms 15 and 16 on Ubuntu 20.04.

I can make it Debian only if that is an issue, but we will be ofuscating the fact that packages will actually work on Ubuntu 21.04+ even though we don't officially support it.

> Will the package also update the locale setting?

No, I forgot about that.

Reply:
> Milestone is not set on this PR. What is the intent?

I set the milestone for the v2021.2. I will try to update the release process asap, if it is blocking can it be done in a fix release?

Reply:
> There is technically no need to go to install the engine / ldns first as it will be installed as dependency of the cli. I will update for users wanting the engine only after this PR as been done (the changes are very similar so once this one is resolved I can port them to engine)

If you start from the main README then you come to Zonemaster-Engine first. That was my point.

Reply:
> I can make it Debian only if that is an issue, but we will be ofuscating the fact that packages will actually work on Ubuntu 21.04+ even though we don't officially support it.

It does not work with Ubuntu as we support Zonemaster. We do not want installations without support for algorithm 15 and 16.

Reply:
>> Will the package also update the locale setting?
>
> No, I forgot about that.

How do you plan to resolve that?

Reply:
> How do you plan to resolve that?

I did not forgot to add it in the package, just in the documentation. In my opinion this should not be done by the package.

Reply:
> It does not work with Ubuntu as we support Zonemaster. We do not want installations without support for algorithm 15 and 16.

Then I will remove the mention for Ubuntu

Reply:
I am fine with having the locale installation outside the package.

Reply:
> Installing with package is also local installation. Make installation with package on Debian the only option for Debian.

I'd prefer not, this won't work for other architecture that amd64 (if someone wants to install the cli on arm64 they would need the cpan instructions)

Reply:
Zonemaster is only supported on "x86_64 / amd64" 

https://github.com/zonemaster/zonemaster/tree/develop#supported-processor-architectures

Reply:
I could be fine to have both package and CPAN installation instructions for Debian, but integrate it into the "local installation".

Reply:
> I could be fine to have both package and CPAN installation instructions for Debian, but integrate it into the "local installation".

Okay, I'll do that. Even if we don't support other architecture, I prefer to keep both at least for this release as this is quite new.

Reply:
Yes, that is a good point.

Reply:
I drafted a release process here: https://github.com/zonemaster/zonemaster/pull/1019